xmipptomo/protocols/protocol_project_top.py

```diff
@@ -154,7 +154,7 @@ def createOutputStep(self):
         # Input could be SetOfVolumes or SetOfSubtomograms
         for item in input.iterItems():
             idx = item.getObjId()
-            p = Particle()
+            p = Particle(objId=item.getObjId())
```

Comment:
If project top is getting absorbed by project subtomograms, shouldn't this protocol get deleted?